websites/docs-smoke-test/src/code-examples/example.yml

```diff
@@ -0,0 +1,10 @@
+- title: Menu
```

Comment:
The file extension here should be `yaml`. Or you add a mapping `yml -> yaml`

https://github.com/commercetools/commercetools-docs-kit/blob/718ea08ac05a8db7c973fef8a3330c5cf15aea97/packages/ui-kit/src/components/code-block.js#L197-L203

Reply:
both are common, I vote for a mapping.